### PR TITLE
Create new home page pitch view

### PR DESCRIPTION
**Pull request by mictab**
screenshot 2018-11-10 at 12 48 38

### CODE REVIEW

File: app/components/HomePage/HomePagePitch/index.js [hunk 1]
```diff
@@ -2,31 +2,65 @@ import React from 'react'
...
 function HomePagePitch() {
   return (
     <div className={styles.pitch}>
       <h1 className={styles.headline}>
         <FormattedMessage {...messages.headline} />
       </h1>
-      <Section image={Project} {...messages.project} />
-      <Section image={Event} {...messages.events} />
-      <Section image={Trip} {...messages.trip} />
+      <Section {...messages.project}>{projectNode()}</Section>
```
Comment (willwull) [inline app/components/HomePage/HomePagePitch/index.js]: I think if we rename `projectNode` to `ProjectNode`, we can instead write:
```js
<Section {...messages.project}>
  <ProjectNode />
</Section>
```
I think that looks a bit nicer maybe?

Comment (mictab) [reply app/components/HomePage/HomePagePitch/index.js]: Yeah, it does. Good suggestion.

Comment (mictab) [reply app/components/HomePage/HomePagePitch/index.js]: Fixed @willwull

**Comment by willwull** [comment]
Also, for the project image, there seems to be some extra whitespace to the left of it:
screenshot 2018-11-10 at 12 58 23
That particular image also seems to shrink a lot when the window width changes, not sure if intentional.

Lastly, I think we should also decrease the padding on mobile, I think this is the responsible code?
```css
@media (orientation: portrait) {
  .section {
    flex-direction: column;
  }

  .section + .section {
    margin-top: 2rem;
  }

  .section div {
    padding: 1rem 2rem; <----------------
  }

  .images {
    flex: 0 1 200px;
    order: 1;
  }

  .text {
    order: 2;
    flex: 100%;
    width: 100%;
    max-width: 100%;
  }
}
```

**Pull request merged**